tools/stats/stats.go

```diff
@@ -120,7 +130,34 @@ func (s *Stats) IteratePullRequests() error {
 	return nil
 }
 
-func (s *Stats) getComments(pr *github.PullRequest) ([]*github.PullRequestComment, error) {
+func (s *Stats) getIssueComments(pr *github.PullRequest) ([]*github.IssueComment, error) {
+	c := s.clientSource()
+	ctx := context.Background()
+	opts := &github.IssueListCommentsOptions{
+		Since: &s.earliestDate,
```

Comment:
This is what we do for `PullRequestComment` too, but it seems wrong - why query comments since this date if we want to have both recent/all counts?

Reply:
Yeah, this does seem like a bug. Do you want to fix it in this PR or should we use another one?